index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
Please kindly set properly the value of `href="css/style.css"` in order to link the stylesheet to the .html file, you can do it by right-clicking on the file and selecting the option copy relative path and paste it 😃 👍